waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -116,6 +120,7 @@ npmDepsForWasp spec =
             ("morgan", "~1.10.0"),
             ("@prisma/client", show prismaVersion),
             ("jsonwebtoken", "^8.5.1"),
+            -- NOTE: secure-password has a package.json override for sodium-native.
```

Comment:
Ideally we'd have a better way to connect the dots here, but a comment should hopefully suffice in meantime.